code/modules/vehicles/cars/car.dm

```diff
@@ -93,3 +94,103 @@
 		M.visible_message("<span class='warning'>[M] is forced into \the [src]!</span>")
 	M.forceMove(src)
 	add_occupant(M, VEHICLE_CONTROL_KIDNAPPED)
+
+
+/obj/vehicle/sealed/car/civ
+	name = "car"
+	desc = "This used to be a pedastrian focused station, and now all these cars are ruining it!." //ok boomer
+	icon_state = "car"
+	max_integrity = 250
+	armor = list("melee" = 70, "bullet" = 40, "laser" = 40, "energy" = 0, "bomb" = 30, "bio" = 0, "rad" = 0, "fire" = 80, "acid" = 80)
+	enter_delay = 20
+	max_occupants = 5
+	movedelay = 1
+	var/obj/item/card/id/linked_id = null
+	var/can_drive = FALSE
+
+/obj/vehicle/sealed/car/civ/generate_actions()
+	. = ..()
+	initialize_controller_action_type(/datum/action/vehicle/sealed/horn, VEHICLE_CONTROL_DRIVE)
+
+/obj/vehicle/sealed/car/civ/Cross(atom/movable/AM)
+	. = ..()
+	if(!isliving(AM))
+		return
+	var/mob/living/L = AM
+	if(!(L.mobility_flags & MOBILITY_STAND) && !L.buckle_lying)
+		for(var/i in return_drivers())
+			if(!ishuman(i))
+				continue
+			var/mob/living/carbon/human/boomer = i
+			boomer.say(pick("DON'T LIE INFRONT OF MY CAR YA FUCKIN KNOB", "GET OUTA DA WAY I'M DRIVIN' HERE", "YOU ARE RUINING MY SUSPENSION CUNT!"), forced="car crash")
```

Comment:
check for age first, say boomer lines if old enough, if young instead say something post-millenials would say